### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by gawrysz**

### CODE REVIEW

**Comment by gawrysz** [comment]
Test this please

**Comment by gawrysz** [comment]
Test this please

File: compilers/camk.in [hunk 1]
```diff
@@ -9,5 +9,5 @@ ifeq ($(PIERNIK_DEBUG), 1)
    F90FLAGS += -ggdb -O0 -ffpe-trap=zero,overflow,invalid -fcheck=all -fno-omit-frame-pointer
    F90FLAGS += -Wall -W -Wextra -pedantic-errors -Wno-unused-function
 else
-   F90FLAGS += -O2 -fstack-arrays
```
Comment (Xarthisius) [inline compilers/camk.in]: It used to have a severe performance impact. If you get random segfaults, you can either increase stack size, or identify code that creates too many temp arrays on stack

Comment (gawrysz) [inline compilers/camk.in]: The difference in performance seems to be of order of 10-20% and happens mostly within RTVD solver (see attached pictures). The crashes with `-fstack-arrays` aren't random - they occur for large local domains and can be avoided by splitting local domain into smaller pieces via AMR_bsize parameter (in which case there is much bigger performance drop and many more arrays are created). IMO it is worth investigating where this difference come from and eliminate it, if possible.
stack_arrays_1
stack_arrays_2

**Comment by gawrysz** [comment]
test this please

**Comment by gawrysz** [comment]
test this please

**Pull request merged**